inst/extdata/TMB_tests/likelihoods/test_likelihoods.R

```diff
@@ -0,0 +1,94 @@
+library(testthat)
```

Comment:
Why don't these get run in the official R tests?

Reply:
Models failed to compile on Ubuntu due to parallelization issues. See recommendation to move tests [here](https://github.com/NOAA-FIMS/FIMS/issues/108#issuecomment-1156583438).

Reply:
Added [Rcpp dnorm unit test](https://github.com/NOAA-FIMS/FIMS/blob/108-feature-pull-tmb-r-style-distributions-into-fims-objective-function/tests/testthat/test-dnorm.R) to testthat. Additional Rcpp tests will be developed for the other distrbutions in sprint 3.